collect.py

```diff
@@ -241,19 +242,29 @@ def run_tests(firefox_driver, chrome_driver):
     chrome_driver.quit()
 
 
-os.environ['PATH'] += os.pathsep + os.path.abspath('tools')
-os.environ['MOZ_HEADLESS'] = '1'
-os.environ['MOZ_HEADLESS_WIDTH'] = '412'
-os.environ['MOZ_HEADLESS_HEIGHT'] = '808'
-firefox_profile = webdriver.FirefoxProfile()
-firefox_profile.set_preference("general.useragent.override", "Mozilla/5.0 (Android 6.0.1; Mobile; rv:54.0) Gecko/54.0 Firefox/54.0")
-firefox_driver = webdriver.Firefox(firefox_profile=firefox_profile, firefox_binary=nightly_bin)
-chrome_options = webdriver.ChromeOptions()
-chrome_options.binary_location = chrome_bin
-chrome_options.add_argument("--no-sandbox")
-chrome_options.add_argument("--headless")
-chrome_options.add_argument("--window-size=412,732")
-chrome_options.add_argument("--user-agent=Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5 Build/M4B30Z) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.83 Mobile Safari/537.36")
-chrome_driver = webdriver.Chrome(chrome_options=chrome_options)
-
-run_tests(firefox_driver, chrome_driver)
+def main(bugs):
+    os.environ['PATH'] += os.pathsep + os.path.abspath('tools')
+    os.environ['MOZ_HEADLESS'] = '1'
+    os.environ['MOZ_HEADLESS_WIDTH'] = '412'
+    os.environ['MOZ_HEADLESS_HEIGHT'] = '808'
+    firefox_profile = webdriver.FirefoxProfile()
+    firefox_profile.set_preference("general.useragent.override", "Mozilla/5.0 (Android 6.0.1; Mobile; rv:54.0) Gecko/54.0 Firefox/54.0")
+    firefox_driver = webdriver.Firefox(firefox_profile=firefox_profile, firefox_binary=nightly_bin)
+    chrome_options = webdriver.ChromeOptions()
+    chrome_options.binary_location = chrome_bin
+    chrome_options.add_argument("--no-sandbox")
+    chrome_options.add_argument("--headless")
+    chrome_options.add_argument("--window-size=412,732")
+    chrome_options.add_argument("--user-agent=Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5 Build/M4B30Z) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.83 Mobile Safari/537.36")
+    chrome_driver = webdriver.Chrome(chrome_options=chrome_options)
+    run_tests(firefox_driver, chrome_driver, bugs)
+
+if __name__=='__main__':
+    bug1, bug2 = bugs[0:len(bugs)//2], bugs[len(bugs)//2:]
+    t1 = threading.Thread(target=main, args=(bug1,))
```

Comment:
Can you use `ThreadPoolExecutor` from `concurrent.futures` instead of the lower level `Thread` from `threading`?

Reply:
Also, add a config variable for the number of threads

Reply:
@marco-c How do you want me to set the config variable? A json file or something else.